GeneralsMD/Code/Tools/WorldBuilder/src/WorldBuilderDoc.cpp

```diff
@@ -471,7 +471,7 @@ AsciiString ConvertFaction(AsciiString name)
 {
 	char oldName[256];
 	char newName[256];
-	strcpy(oldName, name.str());
+	strlcpy(oldName, name.str(), ARRAY_SIZE(oldName));
 	strcpy(newName, "FactionGLA");
 	strlcat(newName, oldName+strlen("FactionFundamentalist"), ARRAY_SIZE(newName));
```

Comment:
`oldName+strlen("FactionFundamentalist")` looks unsafe, when `oldName` is shorter than "FactionFundamentalist".